frontends/web/src/components/accountselector/accountselector.module.css

```diff
@@ -1,6 +1,6 @@
 .balance, .balanceSingleValue{
     margin-left: auto;
-    color: var(--color-gray);
+    color: var(--color-white);
```

Comment:
Is `--color-white` here intentional?

Reply:
I would imagine so.

Otherwise, the colour difference between the text itself and the bg (since bg is gray now) would be a bit too subtle.

Reply:
Actually, nevermind. Color gray seems to look ok:

<img width="689" alt="Screenshot 2023-03-13 at 09 06 12" src="https://user-images.githubusercontent.com/28676406/224642268-46bbed07-dcd9-4aea-a205-0d37f6473985.png">

I'll put it back for now,  but using `color-secondary` :)